File: articles/algorithms-in-media-platforms-and-attention-systems/julia/attention_risk.jl
engagement_pressure = 0.92
creator_impact = 0.88
public_knowledge_impact = 0.78
user_control = 0.44
contestability = 0.42
score = (engagement_pressure + creator_impact + public_knowledge_impact + (1 - user_control) + (1 - contestability)) / 5
println("attention_risk_score=", round(score, digits=4))

File: articles/algorithms-in-media-platforms-and-attention-systems/rust/attention_risk.rs
fn main() {
    let engagement_pressure = 0.92_f64;
    let creator_impact = 0.88_f64;
    let public_knowledge_impact = 0.78_f64;
    let user_control = 0.44_f64;
    let contestability = 0.42_f64;
    let score = (engagement_pressure + creator_impact + public_knowledge_impact + (1.0 - user_control) + (1.0 - contestability)) / 5.0;
    println!("attention_risk_score={:.4}", score);
}
